src/test/java/sirius/biz/importer/format/ValueCheckSpec.groovy

```diff
@@ -0,0 +1,68 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.importer.format
+
+import sirius.kernel.commons.Value
+import spock.lang.Specification
+
+
+class ValueCheckSpec extends Specification {
+
+    def "amount scale check correctly marks non numeric values as invalid"() {
+        when:
+        AmountScaleCheck scaleCheck = new AmountScaleCheck(5, 2)
+        Value number = Value.of("TEST")
+        and:
+        scaleCheck.perform(number)
+        then:
+        thrown IllegalArgumentException
+    }
+
+    def "numbers exceeding the provided precision are correctly marked as invalid"() {
```

Comment:
maybe also add a check which ensures that values outside of the range are detected like 123456